llarp/service/outbound_context.cpp

```diff
@@ -567,7 +567,7 @@ namespace llarp
         // verify source
         if (!frame.Verify(si))
         {
-          LogWarn("signature failed");
+          LogWarn("signature verification failed, T=", frame.T);
```

Comment:
Would prefer to change the `T=` to something descriptive (i.e. `ConvoTag=`) but we can do that across the codebase atomically, per #1632.